### PR TITLE
Issue #3516: Make the 'Formatting options' link less distracting.

### PR DESCRIPTION
**Pull request by jenlampton**
Fixes https://github.com/backdrop/backdrop-issues/issues/3516

Before: 
screen shot 2019-02-03 at 1 08 06 am

After:
screen shot 2019-02-03 at 1 07 08 am

### CODE REVIEW

File: core/themes/seven/css/style.css [hunk 1]
```diff
@@ -694,6 +694,9 @@ fieldset .fieldset-wrapper {
   border-radius: 4px;
   background: transparent;
 }
+.js fieldset.collapsible .fieldset-legend {
+  font-size: 16px;
```
Comment (quicksketch) [inline core/themes/seven/css/style.css]: All other font sizes in Seven are either `em` or `%`, we should be consistent here.

Comment (jenlampton) [reply core/themes/seven/css/style.css]: fixed.

File: core/modules/filter/filter.module [hunk 2]
```diff
@@ -1247,7 +1247,7 @@ function filter_process_format($element) {
   $element['format'] = array(
     '#type' => 'fieldset',
     '#attributes' => array('class' => array('filter-wrapper')),
-    '#title' => t('Formatting options'),
+    '#title' => t('Filtering options'),
```
Comment (klonos) [inline core/modules/filter/filter.module]: Can we please leave this change out for 1.12.2 and stick to only the style changes? We can flesh this out for 1.12.3

Comment (jenlampton) [reply core/modules/filter/filter.module]: yep :)

**Comment by jenlampton** [comment]
trying for a new sandbox.

**Comment by jenlampton** [comment]
Box!

**Comment by jenlampton** [comment]
whoops, forgot to rebase.  trying again.

**Comment by jenlampton** [comment]
Box!!

**Pull request merged**